docs/firefly/verify_download.md

```diff
@@ -8,13 +8,14 @@ keywords:
 - windows
 - macOS
 - linux
+- how to
 ---
 # Verify your Firefly Desktop Download
 
-When you download Firefly Desktop, you may want to verify its authenticity to make sure that you downloaded the correct one from the IOTA Foundation GitHub repository. In this tutorial, you learn how to verify Firefly Desktop downloads.
+When you download Firefly Desktop, you should verify its authenticity to make sure that you downloaded the correct one from the IOTA Foundation GitHub repository. In this how-to guide, you learn how to verify various Firefly Desktop downloads.
```

Comment:
```suggestion
When you download Firefly Desktop, you should verify its authenticity to make sure that you have downloaded the correct one from the IOTA Foundation GitHub repository. In this how-to guide, you learn how to verify various Firefly Desktop downloads.
```

Btw "downloaded" is hellowa akward word, is there a synonym?

Reply:
"Transferred" comes closest but thats not quite good enough, imo.

Reply:
Downloaded a-ok👌